sdk-springboot/sdk-springboot-starters/sdk-springboot-starter/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@artur-ciocanu to your and @ThomasVitale  point .. I think we need to remove `sdk` from all the directories under the sdk-springboot parent one.